spec/acceptance/tic-tac-toe_spec.rb

```diff
@@ -31,23 +22,23 @@ def save_result(result)
   end
 
   it 'allows a human to take a valid turn' do
-    AIPlayer.new(update_grid: UpdateGrid.new(game_state: game_state)).plays(0)
+    AIPlayer.new(update_grid: UpdateGrid.new(game_state: game_state), game_state: game_state).plays(0)
     player_turn(1)
     response = view_grid.execute({})
     grid = response[:grid]
     expect(grid).to eq([[:O, :X, nil], [nil, nil, nil], [nil, nil, nil]])
   end
 
-  it 'returns a win with three Os in a row' do
-    player_turn(1)
-    AIPlayer.new(update_grid: UpdateGrid.new(game_state: game_state)).plays(0)
-    player_turn(2)
-    AIPlayer.new(update_grid: UpdateGrid.new(game_state: game_state)).plays(3)
-    player_turn(4)
-    AIPlayer.new(update_grid: UpdateGrid.new(game_state: game_state)).plays(6)
-    response = view_grid.execute({})
-    expect(response).to eq(grid: [[:O, :X, :X], [:O, :X, nil], [:O, nil, nil]], result: :winner)
-  end
+  # it 'returns a win with three Os in a row', focus: true do
```

Comment:
Commented out test :thinking:

Reply:
Commented out tests are usually a smell - If this is only temporary you can skip it (`xit 'returns a win...`) or just delete it

_but_ "temporary" test skips become permanent far too easily and you should be wary of just skipping a test because it's usually trying to tell you something if it's failing